pybricksdev/cli/__init__.py

```diff
@@ -236,6 +236,114 @@ def is_pybricks_usb(dev):
             await hub.disconnect()
 
 
+class Push(Tool):
```

Comment:
Can we rename this to `Download` (and make the command `download` to be consistent with the existing terminology?